taxi/urls.py

```diff
@@ -53,13 +53,25 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path(
+        "drivers/",
+        DriverListView.as_view(),
+        name="driver-list"
+    ),
+    path(
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
```

Comment:
There are duplicate URL patterns for the driver list and detail views. This can lead to conflicts and unexpected behavior. Ensure each URL pattern is unique.